src/main/java/com/rarchives/ripme/ripper/rippers/EightmusesRipper.java

```diff
@@ -122,14 +123,10 @@ else if (!subUrl.startsWith("http://")) {
                     }
                     try {
                         logger.info("Retrieving full-size image location from " + parentHref);
-                        Thread.sleep(1000);
                         image = getFullSizeImage(parentHref);
                     } catch (IOException e) {
                         logger.error("Failed to get full-size image from " + parentHref);
                         continue;
-                    } catch (InterruptedException e) {
```

Comment:
> Isn't it possible for this exception to be thrown?

Not according to Maven.

> Why remove exception handling?

Because I got a compiler error saying that the exception could not be thrown and it built and seemed to run fine without it.